config/secrets.yml

```diff
@@ -19,9 +19,13 @@
 
 development:
   secret_key_base: 5fc0bc43d7cfb12126b2c9567f35d96d0b384eeb94f3014fb20b12214fb839db11166e7341e2c0acbf650c55d0944b4d05dbf8f47263204397fee769768d27b8
+  gthc_email: 'no-reply@gthc.io'
+  gthc_email_password: 'wDSg7J5Ak4ZJajkK'
```

Comment:
The password should not be public. Maybe let's use ENV for now until we complete our Rails 5.2 update onto `staging`.

Reply:
Fix this